_commands/basics/date.md

```diff
@@ -3,22 +3,32 @@
 
 date
 -------
-TODO: Add documentation for this command by submitting a pull request.
+The date command is used to print out, or change the value of, the system's time and date information
 <!-- one line explanation would go here -->
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$date
+output - Thu Feb 8 16:47:32 MST 2001
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
+Format -- a sequence of characters which specifies how output will appear:
```

Comment:
can u words differently, maybe replacing this line with something like "You can define the output of the date command with the following formatting placeholders"?